### DISCUSSION TITLE
BHT-002-GCLZB time deviation

### DISCUSSION THREAD

**Discussion by aitoralonso** [Koenkk/zigbee2mqtt#5501] [Devices]
Hi, I've installed the BHT-002-GCLZB thermostat, paired with zigbee2mqtt and integrated with Home Assistant without issues.

As initial setup, I manually set the thermostat time with current valid value.

The problem is that as time/days go by, the thermostat time gets deviated from actual value in a very sensitive magnitude.

Anyone else with this zigbee thermostat is facing the same issue? Is there any workaround to align it submitting any message from HA or such?

Thanks in advance!

**Comment by samourai64** [comment]
hello
I have the same problem. After few minutes the device get  deviated. I don't understand why and how to correct this problem
Than'ks for your help

**Reply by nicoxygen** [reply]
same problem

**Comment by AureliRoura** [comment]
Same problem

**Reply by AureliRoura** [reply]
when the device is connected to zigbee the time is set a works fine

**Comment by patduckless** [comment]
Has anyone found a solution to this yet?
